app/controllers/mixins/containers_common_mixin.rb

```diff
@@ -58,6 +58,7 @@ def display_name
   def show_container(record, controller_name, display_name)
     return if record_no_longer_exists?(record)
 
+    @current_page ||= 1
```

Comment:
Anyway, we can change
`(params[:page].nil? ? 1 : params[:page].to_i)`
to something like 
`(params[:page].empty? ? 1 : params[:page].to_i)`